multimedia/Sunshine/Portfile

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem               1.0
+PortGroup                cmake 1.1
+PortGroup                github 1.0
+PortGroup                boost 1.0
+
+# bump revision when changes are made to this file
+revision                 1
```

Comment:
You'll want to start at revision zero for a new port.

Also, in terms of best practices, it's preferable to place this immediately after the `version` line.